tests/sample/ebpf.h

```diff
@@ -17,6 +17,11 @@ ntohs(uint16_t us)
     return us << 8 | us >> 8;
 }
 
+typedef uint8_t mac_address_t[6];
+
+#define ETHERNET_TYPE_IPV4 0x0800
+#define ETHERNET_TYPE_IPV6 0x86dd
```

Comment:
We should be able to just `#include <netiodef.h>`
and not have to redefine these here, since it's part of the Windows devkit just like winsock2.h etc are.

Reply:
The sample eBPF programs would have to netiodef.h in that case. I will try that out later. For now I am resolving this.